src/adapters/Base/index.test.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint camelcase: ["error", {allow: ["camel_case_key"]}] */
```

Comment:
Do we need this? 🤔

Reply:
Removed https://github.com/Lahphim/react-surveyor-challenge/pull/65/commits/2205e8d7ca25692f81cbfca1d73ce09f68d77004